### ISSUE TITLE
develop_freqai docker compose image needs plotly

### ISSUE THREAD

**Issue by tomjrtsmith** [freqtrade/freqtrade#7726]
## Describe your environment

  * Operating system: Ubuntu 20.04
  * Python Version: _____ (`python -V`) 3.10
  * CCXT version: _____ (`pip freeze | grep ccxt`)
  * Freqtrade Version: ____ (`freqtrade -V` or 
  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem:

develop_freqai image, occurs as soon as model training starts - image is 2 days old docker won't give me its version number....

### Steps to reproduce:

  1. _____
  2. _____
  3. _____
  
### Observed Results:

  * What happened?
  * What did you expect to happen?

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

  ```
  freqai2       | 2022-11-10 09:31:51,926 - freqtrade.freqai.base_models.BaseRegressionModel - INFO - -------------------- Done training AUDIO/USDT (83.53 secs) --------------------
freqai2       | 2022-11-10 09:31:52,052 - freqtrade.plot.plotting - ERROR - Module plotly not found
freqai2       |  Please install using `pip3 install plotly`
freqai2       | Traceback (most recent call last):
freqai2       |   File "/freqtrade/freqtrade/plot/plotting.py", line 29, in <module>
freqai2       |     import plotly.graph_objects as go
freqai2       | ModuleNotFoundError: No module named 'plotly' 
  ```

**Comment by xmatthias** [comment]
FreqAI does not depend on plotly in it's default configuration.
Please provide the freqAI part of your configuration.

**Comment by xmatthias** [comment]
will be available with 7147f52e025b3e09d33109a80a02d849e293e52f
